addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -766,8 +767,6 @@ registerModel({
                 peerConnection.close();
             }
             delete this._peerConnections[token];
-            browser.clearTimeout(this._fallBackTimeouts[token]);
-            delete this._fallBackTimeouts[token];
```

Comment:
I guess this is the part I'm mentioning on the other comment? So `_removePeer` is already doing it.